runtime/src/snapshot_utils.rs

```diff
@@ -417,9 +543,9 @@ pub fn archive_snapshot_package(
     Ok(())
 }
 
-pub fn get_snapshot_paths<P: AsRef<Path>>(snapshot_path: P) -> Vec<SlotSnapshotPaths>
+pub fn get_snapshot_paths<P>(snapshot_path: P) -> Vec<SlotSnapshotPaths>
```

Comment:
nit: `get_snapshot_paths -> get_slot_snapshot_paths`, 

and a comment along the lines of, "returns a vector of paths to existing snapshot directories under `snapshot_path`"

Reply:
omg yes. I wanted to do this myself too. I initially tried to change as little as possible for this PR. Happy to make this change!

Reply:
Done!

Reply:
nice, much more readable, thanks :)